addons/account/migrations/9.0.1.1/tests/test_account.py

```diff
@@ -1,13 +0,0 @@
-# coding: utf-8
```

Comment:
ah, another proposal: If you just do an sql update on `ir_module_module` setting state to `to upgrade` for the `full_reconcile` module, the full reconcile module will be treated as being updated. Then you can not touch the init hook at all, and this test will be run